GA/post.py

```
@@ -0,0 +1,1126 @@
+        # Set cyclers and grids
+        for row in range(2):
+            for col in range(2):
+                if row == 1 and col == 1:  # Skip the bottom-right subplot (used for legend)
+                    continue
+                ax[row, col].set_prop_cycle(marker_cycle * color_cycle)
+                ax[row, col].minorticks_on()
+                ax[row, col].grid(which='both')
+                ax[row, col].set_xlabel("Normalised radial coordinate [-]")
+
+        # Plot reference data
+        ax[0,0].plot(reference_blading[stage_idx]["radial_stations"] / reference_blading[stage_idx]["radial_stations"][-1],  # Use the radial location as fraction rather than absolute dimension to remove the effect of different diameters
```

Pylint (3.3.7)

[convention] Line too long (237/100)

(C0301)